packages/ai-jsx/.eslintignore

```diff
@@ -1,3 +1,4 @@
 dist
 jest.config.ts
 scripts/move-cjs-to-dist.js
+src/**/*.test.ts
```

Comment:
We don't want to ignore all our tests. But we do want to do path-specific overrides in `.eslintrc.cjs` to configure the linter to be aware of the tests.

What issue were you seeing that made you do this? I can suggest the config change we need.

Reply:
ESLint was complaining about the mismatch between our tsconfig and this.

ai-jsx:lint: /home/mike/repos/ai-jsx/packages/ai-jsx/src/batteries/docs.test.ts
ai-jsx:lint:   0:0  error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/src/batteries/docs.test.ts` using `parserOptions.project`: <tsconfigRootDir>/tsconfig.json
ai-jsx:lint: However, that TSConfig does not include this file. Either:
ai-jsx:lint: - Change ESLint's list of included files to not include this file
ai-jsx:lint: - Change that TSConfig to include this file
ai-jsx:lint: - Create a new TSConfig that includes this file and include it in your parserOptions.project

Reply:
Ok, you can leave it as-is for now, and I'll revisit later.